docs/source/man.rst

```diff
@@ -308,84 +308,145 @@ As the network infrastructure changes, it might be necessary to delete some sour
+  Optional. The product inspection exclusion. Contains the list of products to exclude from inspection. Valid values are jboss_eap, jboss_fuse, and jboss_brms.
+
+The information in a scan might change as the structure of the network changes. Use the ``qpc scan edit`` command to edit an existing scan to accommodate those changes.
+
+Although ``qpc scan`` options can accept more than one value, the ``qpc scan edit`` command is not additive. To edit a scan and add a new value for an option, you must enter both the current and the new values for that option. Include only the options that you want to change in the ``qpc scan edit`` command. Options that are not included are not changed.
+
+**qpc scan edit --name** *name* --name** *name* --sources=** *source_list* **[--max-concurrency=** *concurrency* **]** **--disable-optional-products=** *products_list*
+
+For example, if a scan contains a value of ``network1sources`` for the ``--sources`` option, and you want to change that scan to use both the ``network1sources`` and ``network2sources`` sources, you would edit the scan as follows:
+
+``qpc scan edit --name=myscan --sources network1sources network2sources``
+
+**TIP:** After editing a scan use the ``qpc scan show`` command to review those edits.
 
 Listing and Showing Scans
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ``qpc scan list`` command returns the summary details for all executed scans. The output of this command includes the identifier, the source or sources, and the current state of the scan.
+The ``qpc scan list`` command returns the summary details for all created scan objects or all created scan objects of a certain type. The output of this command includes the identifier, the source or sources, and any options supplied by the user.
 
-**qpc scan list** **--type=** *(connect | inspect)* **--state=** *(created | pending | running | paused | canceled | completed | failed)*
+**qpc scan list** **--type=** *(connect | inspect)*
 
 ``--type=type``
 
   Optional. Filters the results by scan type. This value must be ``connect`` or ``inspect``. A scan of type ``connect`` is a scan that began the process of connecting to the defined systems in the sources, but did not transition into inspecting the contents of those systems. A scan of type ``inspect`` is a scan that moves into the inspection process.
 
-``--state=state``
+The ``qpc scan job`` command returns the the list of scan jobs or a single scan job associated with a scan object. The output of this command includes the scan job identifiers as well as the status of each job and the results.
 
-  Optional. Filters the results by scan state. This value must be ``created``, ``pending``, ``running``, ``paused``, ``canceled``, ``completed``, or ``failed``.
+**qpc scan job** --name** *scan_name* **--id=** *scan_job_identifier* **--status=** *(created | pending | running | paused | canceled | completed | failed)*
```

Comment:
Let's open a separate issue to remove the requirement on scan_name since we don't strictly need this for the API request.

Reply:
We should move this to the scanning section below.